webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue

```diff
@@ -107,23 +182,24 @@
       </a>
     </div>
 
-    <v-card-text class="peopleCardBody align-center pt-2 pb-1">
+    <v-card-text class="peopleCardBody align-center py-1">
       <a
         :href="url"
         :title="user.fullname"
-        :class="(!user.enabled || user.deleted) && 'text-sub-title' || 'text-color'"
-        class="userFullname text-truncate font-weight-bold d-block">
+        :class="usernameClass"
+        class="userFullname font-weight-bold">
         {{ user.fullname }}
         <span v-if="user.external == 'true' " class="externalFlagClass">
           {{ $t('peopleList.label.external') }}
         </span>
       </a>
-      <v-card-subtitle class="userPositionLabel text-truncate py-0">
+      <v-card-subtitle 
+        class="userPositionLabel text-truncate py-0 mt-0 mt-sm-auto">
```

Comment:
`isMobile` is set to `mdAndDown`, are you sure that we should use `mt-sm-auto`, did you made tests on different breakpoints ? (Desktop & Tablet & Mobile)

Reply:
yes!

Reply:
To make it coherent with each other please, if you use `isMobile = mdAndDown` then you will need to use `mt-lg-auto`, knowing that `isMobile = mdAndDown` should be `xsOnly` instead, or you can rename this variable to `isSmallScreen` instead which corresponds to https://github.com/Meeds-io/platform-ui/blob/develop/platform-ui-skin/src/main/webapp/skin/less/variables.less#L416

Reply:
BTW, because of this incoherence, we have two three dots in `md` & `sm` resolutions :
![image](https://user-images.githubusercontent.com/594824/214230704-daa6d46a-75f7-435a-91c0-5dbdb3898f2b.png)

@srenault-meeds @Imenmakni to recheck on ACC please, thanks.

Reply:
Indeed there is a problem here. @MayTekayaa

Reply:
needed to be retested, no? @MayTekayaa